internal/app/subsystems/aio/store/postgres/postgres.go

```diff
@@ -46,6 +47,23 @@ const (
 	CREATE INDEX IF NOT EXISTS idx_promises_sort_id ON promises(sort_id);
 	CREATE INDEX IF NOT EXISTS idx_promises_invocation ON promises(invocation);
 
+	CREATE TABLE IF NOT EXISTS schedules (
+		id                    TEXT,
+		description           TEXT,
```

Comment:
```suggestion
		desc           TEXT,
```

I think its desc in sqlite